### PR TITLE
Updating copyrights of multiple(batch,deployment,naming,persistence and transaction) modules under appserv-tests/devtests 

### PR DESCRIPTION
**Pull request by kokil12**

### CODE REVIEW

**Comment by kokil12** [comment]
@glassfishrobot Run CI tests please

File: appserver/tests/appserv-tests/devtests/deployment/jsr88/misc/exampleFromJ2EETutorial/cb/jaxrpc/build.properties [hunk 1]
```diff
@@ -1,4 +1,45 @@
...
+  recipient has the option to distribute your version of this file under
+  either the CDDL, the GPL Version 2 or to extend the choice of license to
+  its licensees as provided above.  However, if you add GPL Version 2 code
+  and therefore, elected the GPL Version 2 license, then the option applies
+  only if the new code is made subject to such option by the copyright
+  holder.
+-->
+ 
+
+  This file is referenced by the build.xml file.
```
Comment (vinayvishal) [inline appserver/tests/appserv-tests/devtests/deployment/jsr88/misc/exampleFromJ2EETutorial/cb/jaxrpc/build.properties]: This line was there as a comment previously.

File: appserver/tests/appserv-tests/devtests/deployment/jsr88/misc/exampleFromJ2EETutorial/cb/saaj/build.properties [hunk 2]
```diff
@@ -1,4 +1,45 @@
...
+  recipient has the option to distribute your version of this file under
+  either the CDDL, the GPL Version 2 or to extend the choice of license to
+  its licensees as provided above.  However, if you add GPL Version 2 code
+  and therefore, elected the GPL Version 2 license, then the option applies
+  only if the new code is made subject to such option by the copyright
+  holder.
+-->
+ 
+
+   This file contains properties common to the tutorial examples for SAAJ.
```
Comment (vinayvishal) [inline appserver/tests/appserv-tests/devtests/deployment/jsr88/misc/exampleFromJ2EETutorial/cb/saaj/build.properties]: This line appeared as a comment previously.

File: appserver/tests/appserv-tests/devtests/deployment/jsr88/misc/exampleFromJ2EETutorial/common/build.properties [hunk 3]
```diff
@@ -1,25 +1,66 @@
...
-# The following settings have been commented out because they are performed by the devtests 
-# set-up instead.
-#
-#admin.password=
-#admin.host=localhost
-#admin.user=admin
-#admin.port=4848
-#https.port=1043
+ 
+  The following settings have been commented out because they are performed by the devtests
```
Comment (vinayvishal) [inline appserver/tests/appserv-tests/devtests/deployment/jsr88/misc/exampleFromJ2EETutorial/common/build.properties]: all these lines were commented out previously.

File: appserver/tests/appserv-tests/devtests/deployment/jsr88/misc/exampleFromJ2EETutorial/common/build.properties [hunk 4]
```diff
@@ -1,25 +1,66 @@
...
-#db.root=${j2ee.home}/pointbase
-#db.driver=com.pointbase.jdbc.jdbcUniversalDriver
-#db.host=localhost
-#db.port=9092
-#db.sid=sun-appserv-samples
-#db.url=jdbc:pointbase:server://${db.host}:${db.port}/${db.sid}
-#db.user=pbpublic
-#db.pwd=pbpublic
+domain.resources.port=${http.port}   8080
+ db.root=${j2ee.home}/pointbase
```
Comment (vinayvishal) [inline appserver/tests/appserv-tests/devtests/deployment/jsr88/misc/exampleFromJ2EETutorial/common/build.properties]: Similar to other comments.

**Review by vinayvishal — COMMENTED** [review]
Couple of files had few lines in it commented out originally. I see that CI tests have passed. Can you please see if we are skipping few tests or those commented lines actually have no impact?

**Comment by kokil12** [comment]
@glassfishrobot Run CI tests please

**Review by yaminikb — APPROVED** [review]

**Pull request merged**